src/lib/FormGroupLayout.tsx

```diff
@@ -17,7 +18,7 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
     formState: { errors },
   } = useFormContext();
 
-  const fieldError = errors[name];
+  const fieldError = getDeepValue(errors, name);
```

Comment:
Could you please see if this works? 

https://github.com/react-hook-form/react-hook-form/issues/735#issuecomment-762554374

Would be a little nicer than our own custom implementation :-) 

It's basically just a rexport from the lodash get method: https://lodash.com/docs/4.17.15#get

Reply:
seems to work, nie one many thanks 👍